server/src/main/java/com/linecorp/centraldogma/server/internal/admin/authentication/Token.java

```diff
@@ -29,6 +29,12 @@
 import com.fasterxml.jackson.databind.util.ISO8601Utils;
 import com.google.common.base.MoreObjects;
 
+import com.linecorp.centraldogma.server.internal.admin.model.TokenInfo;
+
+/**
+ * Specifies details of an application token.
+ * This will be replaced with {@link TokenInfo}.
```

Comment:
Currently `Token` class is used to specify an application token, so I will let it `@Deprecated` in the next PR.